CMakeLists.txt

```
@@ -20,6 +20,7 @@ set(srv_files
   srv/SetPayload.srv
   srv/SetSpeedSliderFraction.srv
   srv/SetIO.srv
+  srv/GetVersion.srv
```

As discussed in https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/964, let's rename this.

```suggestion
  srv/GetRobotSoftwareVersion.srv
```